helm/kubernetes-cluster-autoscaler-chart/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: A Helm chart for the cluster autoscaler.
 name: kubernetes-cluster-autoscaler-chart
-version: 0.3.0-[[ .SHA ]]
+version: 0.2.1-[[ .SHA ]]
```

Comment:
The changes to fix the problem are fine. What I do not agree with is the version change here. That looks like a downgrade to upgrade. We should just patch and release a new patch version for that. So it should be `0.3.1`. What do I miss?

Reply:
Like, there may was a `0.2.1` already. Why should we overwrite an older version with a newer change?

Reply:
I get your concern the releases that happened so far were:
`0.1.0`
`0.2.0`
`0.3.0` - current WIP with k8s 1.14

What I am proposing:
`0.1.0`
`0.2.0`
`0.2.1` - fix here for k8s 1.13
`0.3.0` Keep as WIP later to get back to k8s 1.14

The best you can check I believe is quay itself:
https://quay.io/application/giantswarm/kubernetes-cluster-autoscaler-chart?tab=releases

Reply:
Yes I don't think downgrading the version is the right call. If we need to upgrade TCs to use 0-3-stable channel we can do a patch release of cluster-operator.

Once we have VOO we can have a similar approach in charts to operators. So we have multiple git branches. But until then I think a patch release of cluster-operator is the best solution.

Reply:
Okay, how should we go about this:
`0.3.0` - k8s 1.13 with fixed RBAC and no changes to labels
`0.4.0` - k8s 1.14 with fixed rbac and labels for 1.14

Does that make sense?
This is kind of a weird spot because I want to push 1.14 and we just now realized that something broke in an earlier release.

Reply:
With the current design I think that works. You can use 0-3-stable for the patch version and 0-4-stable for the 1.14 WIP version.

Otherwise there is the workaround I mentioned earlier of manually pushing to 0-2-stable but I don't like it. Until we have VOO and a better "gitops" story this is where we are.